src/app/cac-unsubscribe/cac-unsubscribe.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="hero-banner hb-sm">
+  <div class="container">
+    <div class="container-inner">
+      <div class="hero-banner__content">
+        <h1>Unsubcribe from Community Advisory Committee</h1>
+        <p>Please confirm the email you used to sign up to the Community Advisory Committee.</p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<section class="static-content mt-2">
+  <div class="container" id="anchor-point">
+    <div class="spinner-container" *ngIf="loading">
+      <div class="spinner-new rotating"></div>
+    </div>
+
+    <!-- Not subscribed yet -->
+    <div class="row" *ngIf="!success">
+      <main class="mb-0">
+        <form>
+          <div class="form-group mb-2">
+            <p class="control-label">Project</p>
+            <div class="form-control-container">
+              <label>{{projectName}}</label>
+            </div>
+          </div>
+          <div class="form-group">
+            <p class="control-label">Email Address</p>
+            <div class="form-control-container">
+              <input required email class="form-control" type="text" name="emailInput" id="emailInput" [(ngModel)]="emailInput" autofocus />
```

Comment:
One of our upcoming tickets is to make this more accessible with a screen reader. Is it possible to use a `label` element to better provide screen reader support?

Reply:
Good catch!  I cleaned it up a bit better including adjusting for label.  Fixed in #4e99353